eng/native/configurecompiler.cmake

```diff
@@ -162,6 +163,13 @@ elseif (CLR_CMAKE_HOST_UNIX)
       add_linker_flag("-Wl,--gc-sections" DEBUG CHECKED)
     endif ()
   endif(UPPERCASE_CMAKE_BUILD_TYPE STREQUAL DEBUG OR UPPERCASE_CMAKE_BUILD_TYPE STREQUAL CHECKED)
+
+  if(CLR_CMAKE_HOST_BROWSER)
+    # The emscripten build has additional warnings so -Werror breaks
+    add_compile_options(-Wno-unused-parameter)
+    add_compile_options(-Wno-alloca)
+    add_compile_options(-Wno-implicit-int-float-conversion)
```

Comment:
How about `-Wno-error=unused-parameter` etc, insteed? That way we still see the warnings (I think) but they're non-fatal.  Otherwise I'm afraid it might be difficult to discover that this is the place that's suppressing the warnings.

[gcc docs `-Wno-error=`](https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html)